nx-X11/programs/Xserver/xkb/Imakefile

```diff
@@ -40,7 +40,7 @@ XKB_DDXDEFS = XkbServerDefines
    LINTLIBS = ../dix/llib-ldix.ln ../os/llib-los.ln
 
 DEFINES = $(XKB_DDXDEFS) $(NX_DEFINES)
-XKB_DEFINES = -DXKB_BASE_DIRECTORY=\"$(LIBDIR)/xkb\" $(XKB_DISABLE)
+XKB_DEFINES = -DXKB_BASE_DIRECTORY=\"/usr/share/X11/xkb\" -DXKB_BIN_DIRECTORY=\"/usr/bin\" -DXKB_DFLT_RULES_FILE=\"base\" -DXKB_DFLT_KB_LAYOUT=\"us\" -DXKB_DFLT_KB_MODEL=\"pc102\" $(XKB_DISABLE)
```

Comment:
urgh... this looks really hard-coded and hidden... Why not set all those in the main Makefile (I saw some changes like that on your big-1.4.2 branch, so maybe this got changed again. If so, please drop this PR / rebase it with the follow up.